.circleci/config.yml

```diff
@@ -67,9 +67,13 @@ jobs:
               CMD="$(circleci tests split commands.txt)"
               echo "Command: $CMD"
               eval $CMD
+              mkdir pitest-report && mv ./target/pit-reports/*/* pitest-report
+              ls -R | grep ":$" | sed -e 's/:$//' -e 's/[^-][^\/]*\//--/g' -e 's/^/   /' -e 's/-/|/'
             else
               echo "build is skipped ..."
             fi
+      - store_artifacts:
+          path: ./pitest-report
```

Comment:
I would make it simple:
```
✔ ~/java/github/checkstyle/checkstyle [master|✔] 
$ ls -la target/
total 32
drwxrwxr-x  8 rivanov rivanov 4096 Mar 13 06:59 .
drwxrwxr-x 13 rivanov rivanov 4096 Mar 13 06:57 ..
-rw-rw-r--  1 rivanov rivanov    0 Mar 13 06:59 actual.txt
drwxrwxr-x  3 rivanov rivanov 4096 Mar 13 06:57 classes
drwxrwxr-x  4 rivanov rivanov 4096 Mar 13 06:57 generated-sources
drwxrwxr-x  3 rivanov rivanov 4096 Mar 13 06:57 generated-test-sources
-rw-rw-r--  1 rivanov rivanov    0 Mar 13 06:59 ignored.txt
drwxrwxr-x  4 rivanov rivanov 4096 Mar 13 06:57 maven-status
drwxrwxr-x  3 rivanov rivanov 4096 Mar 13 06:58 pit-reports
drwxrwxr-x  4 rivanov rivanov 4096 Mar 13 06:57 test-classes
✔ ~/java/github/checkstyle/checkstyle [master|✔] 
 $ ls -la target/pit-reports/
total 12
drwxrwxr-x 3 rivanov rivanov 4096 Mar 13 06:58 .
drwxrwxr-x 8 rivanov rivanov 4096 Mar 13 06:59 ..
drwxrwxr-x 3 rivanov rivanov 4096 Mar 13 06:59 202003130658
✔ ~/java/github/checkstyle/checkstyle [master|✔] 
$ ls -la target/pit-reports/202003130658/
total 24
drwxrwxr-x 3 rivanov rivanov 4096 Mar 13 06:59 .
drwxrwxr-x 3 rivanov rivanov 4096 Mar 13 06:58 ..
drwxrwxr-x 2 rivanov rivanov 4096 Mar 13 06:59 com.puppycrawl.tools.checkstyle.checks
-rw-rw-r-- 1 rivanov rivanov 1634 Mar 13 06:59 index.html
-rw-rw-r-- 1 rivanov rivanov 5514 Mar 13 06:59 style.css
```

just let circleci to copy whole `pit-reports` folder, there will be always one report.
such artifacts are for maintainers, they know this structure, and will have no problems.
New contributors will never find artifacts, and they will test in on their local, contributors do not need this report that much.

Reply:
or at least keep all files under `target` folder, to keep repo sources clean (it is very good practice, we try to fix all CIs to not put anything to repo to have git status clean after any executions).

Reply:
> just let circleci to copy whole `target/pit-reports/` folder, there will be always one report.
> such artifacts are for maintainers, they know this structure, and will have no problems.
> New contributors will never find artifacts, and they will test in on their local, contributors do not need this report that much.

Oh okay, I was trying to do this(simplifying for user). I'll upload the whole folder.